src/util/specific.py

```diff
@@ -60,6 +61,17 @@ def rename_custom_table(
     )
 
 
+def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
+    rename_module(cr, old_module_name, new_module_name)
+
+    add_to_migration_reports(
+        category="Rename Module",
+        message="The custom module '{old_module_name}' was renamed to '{new_module_name}'. {report_details}".format(
+            **locals()
+        ),
```

Comment:
```suggestion
        message="The custom module %r was renamed to %r. %s" % (old_module_name, new_module_name, report_details)
```

Using `locals()` seems an overkill here.